MakeHist/mumu.C

```diff
@@ -162,15 +154,36 @@ void mumu::Loop(TString sample_name)
                                        //Number of b-jet
                                        Nbjet ++;
                                    }
+                                   else{
+                                       if(jet_pt[NJ] > 30){//non_bjet pT > 30
+                                           //non b-jet pT and eta
+                                           NonbjetpT_hist->Fill(jet_pt[NJ]);
+                                           Nonbjeteta_hist->Fill(jet_eta[NJ]);
+                                           //Save non b-jet information
+                                           TLorentzVector non_bJet;
+                                           non_bJet.SetPtEtaPhiE(jet_pt[NJ],jet_eta[NJ],jet_phi[NJ],jet_E[NJ]);
+                                           non_bJets.push_back(non_bJet);
+
+                                           //Number of non b-jet
+                                           Nnon_bjet ++;
```

Comment:
Why do you count the number of jets when the vectors give you that information for free by the .size() method?